enterprise/tasks.py

```diff
@@ -182,3 +207,133 @@ def send_sso_configured_email(
         )
         LOGGER.exception(message)
         raise exc
+
+
+@shared_task
+@set_code_owner_attribute
+def send_group_membership_invitation_notification(
+    enterprise_customer_uuid,
+    group_membership_uuid,
+    budget_expiration,
```

Comment:
I would genericize the name of this parameter a little more. The edx-enterprise code has no notion or definition of budgets or policies.  Maybe something like `act_by_date`?